File: fabric/src/main/java/com/tnt_man_inc/jarm_3/fabric/ExampleExpectPlatformImpl.java
package com.tnt_man_inc.jarm_3.fabric;

import com.tnt_man_inc.jarm_3.ExampleExpectPlatform;
import net.fabricmc.loader.api.FabricLoader;

import java.io.File;

public class ExampleExpectPlatformImpl {
    /**
     * This is our actual method to {@link ExampleExpectPlatform#getConfigDirectory()}.
     */
    public static File getConfigDirectory() {
        return FabricLoader.getInstance().getConfigDir().toFile();
    }
}

File: fabric/src/main/kotlin/com/tnt_man_inc/jarm_3/fabric/ExampleExpectPlatformImpl.kt
package com.tnt_man_inc.jarm_3.fabric

import com.tnt_man_inc.jarm_3.Jarm.init
import net.fabricmc.api.ModInitializer
import net.fabricmc.loader.api.FabricLoader
import java.io.File

object ExampleExpectPlatformImpl {
    /**
     * This is our actual method to [ExampleExpectPlatform.getConfigDirectory].
     */
    @JvmStatic
    fun getConfigDirectory(): File {
        return FabricLoader.getInstance().configDir.toFile()
    }
}